modules/products/confluence/local_home_storage.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_ebs_volume" "local_home" {
+  count             = var.local_home_snapshot_id != null ? var.replica_count : 0
+  availability_zone = var.eks.availability_zone
+  snapshot_id       = var.local_home_snapshot_id
+  size              = tonumber(regex("\\d+", var.local_home_size))
```

Comment:
Would be possible to get the size from the snapshot instead? I.e. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ebs_snapshot#volume_size

What will happen if vendor set the `local_home_size` smaller to the snapshot size?

Reply:
Nice update. LGTM.